dev-util/clue/clue-2.2.0.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="A GUI tracing tool"
+DESCRIPTION="Clue is a comprehensive GUI tracing tool, allowing inspection on almost all of the haiku native objects."
```

Comment:
Keep the DESCRIPTION up to a max of 80 characters

Reply:
[`DESCRIPTION`](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description)